### PR TITLE
Initial overhaul of 28-1900

### PR DESCRIPTION
**Pull request by micahchiang**
# New schema

department-of-veterans-affairs/va.gov-team#12420

This pull request overhauls the existing schema for VRE form 28-1900. 

## Pull Requests to update the schema in related repositories

https://github.com/department-of-veterans-affairs/vets-website/pull/14457 (in draft)
https://github.com/department-of-veterans-affairs/vets-api/pull/5018 (in draft)

### CODE REVIEW

File: test/schemas/28-1900/schema.spec.js [hunk 1]
```diff
@@ -1,67 +1,12 @@
-import SchemaTestHelper from '../../support/schema-test-helper';
-import schemas from '../../../dist/schemas';
-import SharedTests from '../../support/shared-tests';
+// import SchemaTestHelper from '../../support/schema-test-helper';
```
Comment (micahchiang) [inline test/schemas/28-1900/schema.spec.js:1]: Updates to this test file will be included in a separate pull request.

**Review by micahchiang — COMMENTED** [review]
Unit test updates for this schema will come in a separate pull request.

**Review by rianfowler — APPROVED** [review]

**Comment by rianfowler** [comment]
This should get a peer review by someone who has more context.

**Comment by micahchiang** [comment]
> This should get a peer review by someone who has more context.

I checked in `vets-website` and this schema wasn't originally used anywhere. Seems to be an orphaned project as far as I can tell. Checked in `vets-api`, and it isn't referenced in there anywhere either.
